twig/BS3/fields/toggleYesNo.twig

```diff
@@ -1,14 +1,25 @@
 {% macro render(field) %}
-    <div>
-        <div class="btn-group" id="{{ field.id }}" data-toggle="buttons">
+    <fieldset>
```

Comment:
These inline overrides like this cause duplication. We most likely don't want every heading twice.